Dockerfile

```diff
@@ -114,7 +114,7 @@ RUN if [ "$USE_TARGETID" = "OFF" ] ; then echo "MIOpenTensile is not installed."
 
 RUN if [ "$USE_MLIR" = "ON" ]; \
     then cd ~ && \
-    export MLIR_COMMIT=199d667b9d8caaf283436aaa8a48fd20e074f42c && \
+    export MLIR_COMMIT=31579e0c5cf6eb4d7b1db0d349407f8bab547d9b && \
```

Comment:
Note: Bumping the commit because we recently fixed a number of tuning related failures. No interface change therefore no need to make any MIOpen implementation updates.